projects/packages/search/src/class-rest-controller.php

```diff
@@ -268,6 +269,13 @@ public function update_settings( $request ) {
 			}
 		}
 
+		if ( $swap_classic_to_inline_search !== null ) {
+			$swap_classic_to_inline_search_updated = $this->search_module->update_swap_classic_to_inline_search( $swap_classic_to_inline_search );
+			if ( is_wp_error( $swap_classic_to_inline_search_updated ) ) {
```

Comment:
Will we get to that at any point? I wouldn't expect `update_option` to run a `WP_Error`.

Reply:
No, you're right, `update_option` won't return an error. The other methods in this file had some custom checks, but we didn't need it here, so I've removed this `is_wp_error` check.